Resources/config/services.xml

```diff
@@ -199,6 +199,38 @@
             <argument type="service" id="fos_rest.view_handler"/>
         </service>
 
+        <!-- Form Controller -->
+        <service id="sulu_form.form_controller" class="Sulu\Bundle\FormBundle\Controller\FormController" public="true">
+            <argument type="service" id="fos_rest.view_handler.default"/>
+            <argument type="service" id="security.token_storage"/>
+            <argument type="service" id="sulu_form.manager.form"/>
+            <argument type="service" id="sulu_core.doctrine_rest_helper"/>
+            <argument type="service" id="sulu_core.doctrine_list_builder_factory"/>
+            <argument type="service" id="sulu_core.list_builder.field_descriptor_factory"/>
+            <argument type="service" id="sulu_core.list_rest_helper"/>
+
+            <tag name="sulu.context" context="admin"/>
+        </service>
+
+        <!-- Form Website Controller -->
+        <service id="sulu_form.form_website_controller" class="Sulu\Bundle\FormBundle\Controller\FormWebsiteController" public="true">
+            <argument>%sulu_form.ajax_templates%</argument>
```

Comment:
should not be needed as `parameter_bag` is injected so you can use `$this->getParameter` function to get them